doc/contributing/pull-requests.md

```diff
@@ -511,12 +510,14 @@ feedback.
 All pull requests that contain changes to code must be run through
 continuous integration (CI) testing at [https://ci.nodejs.org/][].
 
-Only Node.js core collaborators with commit rights to the `nodejs/node`
-repository may start a CI testing run. The specific details of how to do
-this are included in the new collaborator [Onboarding guide][].
+Only Node.js core collaborators and triagers can start a CI testing run. The
+specific details of how to do this are included in the new collaborator
+[Onboarding guide][]. Usually, a collaborator or triager will start a CI
+test run for you as approvals for the pull request come in.
+If not, you can ask a collaborator or triager to start a CI run.
 
 Ideally, the code change will pass ("be green") on all platform configurations
-supported by Node.js (there are over 30 platform configurations currently).
+supported by Node.js. (There are over 30 platform configurations currently.)
```

Comment:
IMO it's strange to have a whole sentence as a parenthetical (and also... off to have currently at the end of that parenthetical). Two potential alternativeS:

```suggestion
Ideally, the code change will pass ("be green") on all 30+ platform
configurations supported by Node.js.
```

```suggestion
Ideally, the code change will pass ("be green") on all platform configurations
supported by Node.js. There are currently over 30 platform configurations.
```

Reply:
I went with Option 3: Delete the sentence entirely because it does not contain essential information. PTAL.